sql/moz-fx-data-shared-prod/telemetry_derived/experiment_search_aggregates_v1/query.sql

```diff
@@ -135,6 +135,43 @@ fenix AS (
     experiment,
     branch
 ),
+ios AS (
+  SELECT
+    submission_timestamp,
+    experiment.key AS experiment,
+    experiment.value.branch AS branch,
+    SUM(0) AS ad_clicks_count,
+    SUM(0) AS search_with_ads_count,
+    SUM(
+      (SELECT SUM(value.value) FROM UNNEST(metrics.labeled_counter.search_counts) AS value)
+    ) AS search_count,
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_ios_firefox_stable.metrics_v1`
+  LEFT JOIN
+    UNNEST(ping_info.experiments) AS experiment
+  GROUP BY
+    submission_timestamp,
+    experiment,
+    branch
+  UNION ALL
```

Comment:
In `experiment_enrollment_aggregates`, we union at the ping level and then aggregate in a subsequent step, which reduces code duplication for the aggregation. Is there a reason that we can't apply the same treatment here?

Reply:
The structure of `metrics` is different, for example, for the different fenix pings. So we can't directly union them but instead have to select the specific metrics we care about which means we have to do some (duplicated) `UNNEST`ing here.